cmd/go2gql/main.go

```diff
@@ -36,6 +36,12 @@ func main() {
 			if err != nil {
 				panic(err)
 			}
+
+			err = gc.ParseImports()
+			if err != nil {
```

Comment:
probably, inline declaration should be better?

Reply:
```suggestion
			if err := gc.ParseImports(); err != nil {
			     return errors.Wrap(err, "failed to parse imports")
			}
```

Reply:
done